### ISSUE TITLE
Footer list items focus ring contrast

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#73]
**Pagina:**
[Zoek API's Template](https://rijkshuisstijl-community-templates-eviwhsy01-nl-design-system.vercel.app/developer-overheid/api-overzicht)

1. **Probleembeschrijving:**
De focusring van de items in de Footer heeft niet genoeg contrast met de achtergrond.

**Referentiecode:**
`<span class="utrecht-link-list__link-text">Contact</span>`

**Referentieafbeelding:**

<img width="156" alt="A focused link with the text Contact, has a black focus ring against a dark blue background" src="https://github.com/user-attachments/assets/109d5cf2-3b2e-4e21-888f-7592ca2ccfd3" />
